storybookjs/storybook
# Addon-docs: Add inline rendering for svelte

Issue:

## What I did
i added inline rendering support for svelte components, i used a HOC.svelte file to mount the components passed into StoryFn() on the app. 
## How to test

run storybook on svelte-kitchen-sink to see the inline components.

If your answer is yes to any of these, please make sure to include it in your PR.
